src/docs/product/relay/operating-guidelines.mdx

```diff
@@ -113,3 +114,72 @@ limits:
 ```
 
 See the [Configuration Options](/product/relay/options/) page for detailed descriptions of all available options.
+
+## Performance Tuning
+
+Relay has quite a lot of different [Configuration Options](/product/relay/options/), and
+changing some of them can have more impact on Relay's behavior than others.
+
+Here we list a few relevant options that can help you to tune Relay to specific environment and workload:
+
+- `limits.max_concurrent_requests` (default: 100)
+
+  Number of concurrent requests your Relay instance can send to
+  the upstream (Sentry). If your event volume or connection latency to Sentry are high, it might be
+  beneficial to increase this value to gain additional throughput at the expense of
+  additional network connections.
+
+- `cache.event_buffer_size` (default: 1000)
+
+  How many events Relay can buffer in its local queue before it starts rejecting new events.
+  Increasing this value will also increase Relay's potential memory consumption when, for
+  example, there are network issues that prevent Relay from forwarding the received messages to Sentry.
+
+- `cache.event_expiry` (in seconds, default: 600)
+
+  How long Relay can keep buffered events in memory before dropping them. It is worth
+  increasing this value if you foresee a situtation when your Relay has to keep events in
+  memory for longer than the default value.
+
+- `cache.project_expiry` (in seconds, default: 300)
+
+  To stay operational, Relay regularly fetches project configurations from the Sentry
+  upstream. This setting controls how often Relay does the fetch, and can be reduced to
+  speed up the configuration propagation. For instance, if you change data scrubbing options
+  in your project settings in Sentry, those will be propagated to your Relay instance faster.
```

Comment:
```suggestion
  To stay operational, Relay regularly fetches project configurations from the Sentry
  upstream. This setting controls how often Relay fetches that configuration. You can reduce the frequency to speed up the configuration propagation. For example, if you change data scrubbing options in your project settings in Sentry, those will be propagated to your Relay instance faster.
```

The last sentence is a bit confusing - do we mean that the settings will be propagated to Relay faster, or do we mean changing data scrubbing in project settings will speed up Relay?

Reply:
The data scrubbing configuration will be propagated to Relay faster.
So by reducing this value we _increase_ the frequency of those requests, e.g. fetching will be happening every minute, instead of every five minutes. It's basically the worst-case-scenario time that Relay has to wait to start using the configuration that was updated in Sentry.

What do you think about this:

```suggestion
To stay operational, Relay regularly fetches project configurations from the Sentry upstream. This setting controls how often Relay fetches that configuration. You can decrease this value to make the configuration propagation more frequent. For example, if you later change data scrubbing options in your project settings in Sentry, your Relay instance will become aware of those changes faster.
```

Reply:
Yes, this is much more clear.

Reply:
Thanks!